x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/translations.ts

```diff
@@ -75,8 +75,8 @@ export const VIEW_ALL_FAILED_FINDINGS = i18n.translate('xpack.csp.view_all_faile
   defaultMessage: 'View all failed findings',
 });
 
-export const RESOURCE_TYPE = i18n.translate('xpack.csp.resource_type', {
-  defaultMessage: 'Resource Type',
+export const CIS_SECTION = i18n.translate('xpack.csp.cis_section', {
```

Comment:
the message id can use more subsections to follow the i18n guidelines more closely. a suggestion would be:
`xpack.csp.dashbaord.risksTable.cisSectionColumnLabel`

Reply:
done